itest.rb

```diff
@@ -1,7 +1,10 @@
 class Unknown
   def initialize(text)
     # Initalize hash    
-    @words = Hash.new
+      @words = Hash.new # two space for indentation
+      
+    #Don't use ; to separate statements
+    puts 'foo'; puts 'bar' # two expressions on the same line  
```

Comment:
__[rubocop]__
* Inconsistent indentation detected.
* Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)